packages/nuxt/src/vite/addServerConfig.ts

```diff
@@ -94,7 +94,6 @@ export function addDynamicImportEntryFileWrapper(nitro: Nitro, serverConfigFile:
   }
 
   nitro.options.rollupConfig.plugins.push(
-    // @ts-expect-error - This is the correct type, but it shows an error because of two different definitions
```

Comment:
@s1gr1d I guess this was probably due to rollup version "mismatches" then and it picking up types differently from the rollup version used by nuxt 🤔